### ISSUE TITLE
[Bug] Pinch-zoom in webtoon view causes fast random scrolling

### ISSUE THREAD

**Issue by geominorai** [Jays2Kings/tachiyomiJ2K#115]
### Device information
* Tachiyomi version: release 0.9.81
* Android version: 9
* Device: Samsung Galaxy S9+

## Steps to reproduce
1. Enable webtoon viewing mode
2. Pinch to zoom in or out

### Expected behavior
Simple change of magnification without scrolling

### Actual behavior
1/3 of the time it scrolls to high heaven (multiple pages away) as if flicking scrolling.

### Other details
Video with Android dev option touch trails visible: https://www.dropbox.com/s/j71zz09hs8aob34/Tachiyomi_2020-02-16-19-34-38_1.mp4?dl=0

See also same issue reproduced on original Tachiyomi: https://github.com/inorichi/tachiyomi/issues/2602

**Comment by geominorai** [comment]
Following an OTA upgrade to Android 10, my S9+ no longer scrolls extreme distances following pinch release.  Now there is only some scrolling in the direction of the last released finger press, which is a bit annoying but a lot more tolerable.

But this suggests the problem may be the app not waiting a bit following the start of the pinch release to allow both fingers to lift before starting to recognize a scroll gesture.

**Comment by geominorai** [comment]
This bug is still present in version 1.0.1: [Video](https://i.imgur.com/n9xbBL0.mp4)

**Comment by joseph619** [comment]
Does this still happen?

**Comment by geominorai** [comment]
> Does this still happen? 

Yes, even on the current release, it still happens for me multiple times a day, so much so that it is very annoying.

**Comment by geominorai** [comment]
Or rather the underlying bug anyhow. It's also very reproducible with double-tap zoom.
